src/api/app/views/webui/search/_results_issue.html.haml

```diff
@@ -21,4 +21,4 @@
         - else
           = truncate(result.description, length: 80)
   - if per_page
-    = paginate results
+    = paginate results, , views_prefix: 'webui'
```

Comment:
What is the second comma for?

Reply:
Ups, it's a typo I probably introduced right after testing the view. Thanks!!